### PR TITLE
feat(cb2-14233): Update pull-test-results/modify to remove snowball SQS pattern

### PR DESCRIPTION
**Pull request by m-mullen**
## Update pull-test-results/modify to remove snowball SQS pattern
Update pull-test-results/modify to remove snowball SQS pattern
[CB2-14233](https://dvsa.atlassian.net/browse/CB2-14233)

## Checklist
- [x] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/insert.ts [hunk 1]
```diff
@@ -11,18 +11,12 @@ logger.debug(
...
+  const batchItemFailures = await eventHandler(event);
 
-    logger.info('Data processed successfully.');
-    callback(null, 'Data processed successfully.');
-  } catch (error) {
-    logger.info('Data processed unsuccessfully.');
-    logger.error('', error);
-    callback(new Error('Data processed unsuccessfully.'));
-  }
+  logger.info(`Data processed successfully with ${batchItemFailures.batchItemFailures.length} failures`);
```
Comment (Daniel-Searle) [inline src/insert.ts:18]: I don't think that we need to have this log

**Review by Daniel-Searle — APPROVED** [review]

File: src/insert.ts (hunk 1 above)
Comment (naathanbrown) [reply src/insert.ts:18]: I'm happy with this either way

**Review by naathanbrown — APPROVED** [review]

File: src/insert.ts (hunk 1 above)
Comment (cb-cs) [reply src/insert.ts:18]: I'm not fussed, but either way I think both handlers should be consistent. So either they both have it, or neither have it.

**Comment by cb-cs** [comment]
Code looks good but there is no coverage for the 2 lines below in the eventHandler catch block, can these be added please. Happy to approve then.

```
console.error(error);
batchItemFailures.push({ itemIdentifier: record.messageId });
```

**Review by cb-cs — APPROVED** [review]

**Pull request merged**